### ISSUE TITLE
matsim highly negative scoring  even though plans are correctly fulfilled

### ISSUE THREAD

**Issue by federicobigi** [matsim-org/matsim-code-examples#696]
Hello, I'm creating a new population for a designed network. I don't know why, but apparently, when I run matsim multiple agents tend to have a very low scoring even though they correctly fulfill their plans on time. For example, I isolated two agents and run them in an empty network, they finish the simulation matching exactly their end_time for each activity but still their score is respectively -450 and -910. I think it could be related to the config file, but I'm not sure where I am mistaking it.
I'll attach here the initial plan, the experienced plans, and the config file. If you need any other information please tell me
[data.zip](https://github.com/matsim-org/matsim-code-examples/files/8214965/data.zip)

**Comment by kainagel** [comment]
This looks all quite reasonable.  Possibly the short home and/or work durations cause problems.  In particular the short home duration starting 9:24:05.  You could use something like
```
			<parameterset type="activityParams" >
				<param name="activityType" value="home" />
				<param name="scoringThisActivityAtAll" value="false" />
			</parameterset>
```
to switch off the activity scoring for the "home" activity (or any other activity type).  If this is indeed the problem, a way to proceed would be to define activities home_1800, home_3600, home_7200 etc. for home activities of typical length 1800, 3600, 7200 etc. seconds.  

(I am a bit surprised that this is not the case in the Velbert scenario since this is now our standard approach, but I also did not check in detail.)

PS: Thank you for sending complete and reduced information.

**Comment by federicobigi** [comment]
[data_2.zip](https://github.com/matsim-org/matsim-code-examples/files/8363434/data_2.zip)
Thanks for your reply and sorry for my late reply.
At first, setting the parameters <param name="scoringThisActivityAtAll" value="false" /> showed me that actually, as you identified clearly, the problem lies within the activities being too small. 
The problem that I have with my population is that is extracted from a detailed survey, and therefore I'm afraid that if I force the activities to a fixed amount of time this would change the time distribution of the other activities of the agent, and this would maybe end up in some of missing some pts. I, instead, tried this workaround that you will find attached, but I am not super sure if it is the correct way to proceed, and therefore I'd like to hear your opinion on this because the output shows that the scoring becomes positive, but I'm not sure if this could impact the scoring in other unseen directions.
Basically, instead of changing the activities and splitting them into the multiple x_1800, x_3600 ... , I set _minimalDuration_ = 00:01:00, _latestStartTime_ to 24:00:00 and the _typicalDuration_ = 00:01:00 of the activities that were causing troubles. This apparently solved the problem.
But, even with the book, this is a thing that is still not clear to me, especially on this last parameter: is actually typicalDuration impacting on the scoring of the agent? Because on the other two is clear in the book, about this one i am not sure.
Thanks a lot for the attention and wishing you all a nice day.
